source/compiler/sc2.c

```diff
@@ -2090,7 +2090,7 @@ char *sc_tokens[] = {
          "assert", "*begin", "break", "case", "char", "const", "continue", "default",
          "defined", "do", "else", "emit", "__emit", "*end", "enum", "exit", "for",
          "forward", "goto", "if", "native", "new", "operator", "public", "return",
-         "sizeof", "sleep", "state", "static", "static_assert", "static_check", "stock",
+         "sizeof", "sleep", "state", "__static", "__static_assert", "static_check", "stock",
```

Comment:
@YashasSamaga Probably not what you really intended...

Reply:
Yup, realized it while writing the test. Thanks.

Reply:
What's worse is that the `WILL_FAIL` testing method does not work with regular expression tests (which means pretty much all the merged tests for errors are wrong)